tests/ssl-opt.sh

```diff
@@ -1428,11 +1428,13 @@ run_test    "Renegotiation: server-initiated" \
             -c "found renegotiation extension" \
             -c "=> renegotiate" \
             -s "=> renegotiate" \
-            -s "write hello request"
+            -S "write hello request" \
+            -S "client hello v3, signature_algorithm ext: 2" # Is SHA-1 negotiated?
```

Comment:
I think comments only work at the start of lines, based on what I see in `tests/suites/main_test.function`.

Reply:
@Patater At least on my machine it appears to work.

Reply:
@Patater `test/suites/main_test.function` is only for generating the test suite source files. It's not related to `ssl-opt.sh` to my knowledge.

Reply:
Here comments work at the end of the line too, because`tests/ssl-opt.sh` is a plain bash script, whereas `tests/suites/main_test.function` is used by a test suite generation script to compile it to C source code.

Reply:
Yay, I like being able to have comments.

OK